app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Please modified the `login` to match the wireframes and their styling.
![login](https://user-images.githubusercontent.com/55596970/186610083-d913010e-5f77-421f-bcdd-284f1134d803.png)

Reply:
Hello @CollinsTatang .

The wireframe does not include both the "Forgot your password?" and the "Sign up" link.
Does that mean the user is expected to know/guess the URL for those required functionalities?

Please confirm what should I do to those links. Should I ommit them?

Reply:
Hi @rbanos-mv 
As per project requirement, you are expected to follow the [Wireframe Design](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)

Reply:
You can check the last point on your [ project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md)